exonum/src/blockchain/config.rs

```diff
@@ -123,6 +123,15 @@ pub struct ConsensusConfig {
     pub propose_timeout_threshold: u32,
 }
 
+/// Service parameters.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ServiceConfig {
+    /// This service is considered enabled.
+    pub enabled: bool,
+    /// Opaque service-specific configuration serialized as JSON.
+    pub private: serde_json::Value,
```

Comment:
I also think `private` is not a very good name. Maybe, something like `details` can be better?